src/gui/EQ/EQPlot.cpp

```diff
@@ -240,6 +280,12 @@ void EQPlot::setSelectedBand (int bandIndex)
 
 void EQPlot::paint (juce::Graphics& g)
 {
+    const float maxGain = 18.0f;
+    gui::eq::drawDbText (g, *this, maxGain);
+    gui::eq::drawFrequencyText (g, *this, 100, -maxGain);
+    gui::eq::drawFrequencyText (g, *this, 1000, -maxGain);
+    gui::eq::drawFrequencyText (g, *this, 10000, -maxGain);
```

Comment:
I wonder if it might be better to have the frequency labels at the top of the screen so the chyron doesn't get in the way? I'm not sure though... having the labels at the bottom probably looks a little bit neater.

Reply:
If we put them up top 10kHz is partially obscured by the drawButton, I'm ok with either but it seems the bottom is more standard practice?
<img width="728" alt="Screenshot 2023-06-02 at 22 25 11" src="https://github.com/Chowdhury-DSP/ChowMultiTool/assets/40403174/48e9355d-e73d-4642-aa69-5a011e0a126c">